### PR TITLE
Project-3151

### PR DESCRIPTION
**Pull request by Eagletanker**
## About the PR 
You know, I thought of just copy pasting a picture of some space age propganda, but I dont think that would do it justice. This is a prefab that adds in a ship that is caught in X0s outer asteroid belt.

BIG THANKS TOO
Cogwerks
Yellow
Dimwhat
Walp
Batelite
End
Gannets
And RP-chat for dealing with my bullshit related to this

Without those wonderful people, this would have never been completed nor even made to begin with. Details are vauge on purpose, arm yourself with your trusty Gaggle Translate and your toolbox, and head out to the mining belt to learn what it is!
(Note, I am sorry to any and all russian speakers for whatever fucked translation will come from the notes. If you know russian and can actually provide a accurate translation for this, please let me know on discord, and I'll send you the raw notes.)
## Why's this needed? 
More stuff good. It also answers some questions people might have, and gives a jumping off point for some Fun (TM) stuff.

## Changelog

```changelog
(u)Eagle
(+) A ship of unknown orgin and construction has drifted into the mining belt.
```

### CODE REVIEW

**Comment by Eagletanker** [comment]
Ah fuck I need to clean this I think

**Comment by Zonespace27** [comment]
Added review requests as per author's wishes

**Comment by Eagletanker** [comment]
The runtimes the checker is reporting are not showing up on my local, will keep investigating though

**Comment by Eagletanker** [comment]
Might have fixed it, lets see

**Review by Sovexe — CHANGES_REQUESTED** [review]
Messes up z2 by deleting a whole column of the map and sliding it over, also makes changes to admin offices.

All of these erroneous changes will need to be reverted

image

**Comment by Eagletanker** [comment]
> Messes up z2 by deleting a whole column of the map and sliding it over, also makes changes to admin offices indicating merge conflicts that were not properly resolved.
> 
> All of these erroneous changes will need to be reverted. I'd highly recommend copying your changes out and just repasting onto a clean copy of z2.
> 
> image

Fixed them all

**Review by flrsh — CHANGES_REQUESTED** [review]
In order for devs to review this, you should include screenshots of the entire area and a detailed description of what you're actually adding. The description should include what this actually is, any notable loot, and why we should add this. If you're concerned about spoilers, put all of this in a doc or something and DM it to one of us. 

And if you're adding notes in another language, please make sure the translation quality is better than Google Translate.

Also seeing some weird duplicate additions in `bullet.dm`.

File: code/z_adventurezones/hospital.dm [hunk 1]
```diff
@@ -461,6 +461,75 @@ var/list/hospital_fx_sounds = list('sound/ambience/spooky/Hospital_Chords.ogg',
 	mat_changename = 0
 	mat_changedesc = 0
 
+/obj/soviet/setpiece/console/nuclearcontrol //fuck you, its 133 at night and I'm tired cause it wont compile so this goes here. Fuck. Also there doesnt seem to be any other large grouping of soviet shit and fun there is really no organization for factional things I guess. Anyways I'm also gonna shove notes in here now.
```
Comment (ZeWaka) [inline code/z_adventurezones/hospital.dm:464]: not really acceptable

File: code/z_adventurezones/hospital.dm [hunk 2]
```diff
@@ -461,6 +461,75 @@ var/list/hospital_fx_sounds = list('sound/ambience/spooky/Hospital_Chords.ogg',
...
+	icon_state = "sovbank2"
+	desc = "Some sort of plotting chart. Could also be one of those graph things you see in the pictures. You know the ones."
+
+/obj/soviet/setpiece/console/bank2 //again, thank you cog so very much for the sprites
+	name = "Monitor Bank"
+	icon = 'icons/obj/large/64x64.dmi'
+	icon_state = "sovbank1"
+	desc = "A wall of monitors, used for something. It doesnt matter much anymore though."
+
+/obj/item/paper/thermal/sovietreport1 //premtively I am sorry to any russians or people who speak russian that play this game.
```
Comment (ZeWaka) [inline code/z_adventurezones/hospital.dm:488]: see: flrsh's translation comment

**Review by ZeWaka — CHANGES_REQUESTED** [review]
* not a big fan of the power conduits being used as... wings?
* the airlocks(?) look pretty atrocious
* so many of these rooms are just empty with nothing in them - not even a spec of dirt
* why is this so big - there's 4 copypasted rooms - it looks like you made the outline and then tried to fit stuff inside it

File: code/obj/machinery/door/airlock.dm [hunk 3]
```diff
@@ -583,6 +583,12 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 	welded_icon_state = "2_welded"
 	req_access = null
 
+/obj/machinery/door/airlock/soviet
```
Comment (ZeWaka) [inline code/obj/machinery/door/airlock.dm:586]: you set the layer manually on every instance of this.

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Comment by Eagletanker** [comment]
> 

In reference to this

The power cables here are actually radiators, using the same sprite as on TEG maps. This is out of the lack of a better sprite.

Airlocks I'll work on more, it'll take me a bit to make them look good with animation and the way they move. I'm aiming for something distinct from NT and the Syndicate, same as the current WIP Mars airlocks that can be seen in Imspriter at the moment, and with the old non persective airlocks. They were also manually assigned a layer on purpose, I was making sure the values I put in would work and overlay above players. Will work on improving readability there.

1. There's alot of dirt so I dont know if that sarcasm or not
2. The top level is there as a way to introduce and give people a safe spot in the prefab before descending into it. The bottom layer is more heavily decorated, and I'll add more if needed.

The design I settled on as it looks good and pretty plausible to actual combat ships. It also draws from the soviet Sci Fi Magazine TM (I forget the full name). There's inferences to be made once you go down to the lower level as to why the top is so sparse. I'll expand upon that and draw more attention to the ladders though. It seems that was not easy to tell that you were supposed to go down them. 


Bullet.dm will be fixed soon, and a pitch document has been sent to the developers.

**Comment by Eagletanker** [comment]
Gonna close this for now. After leaving it in the drawer for a bit and looking over it, there's some more stuff I want to do. Namely
Formal Naval outfits
Remapping the top portion cause as it stands the entire crew gets killed by the radiation from the reactor. 
Improving the airlock animation (Probably will make a iris design that's stronger)
Few other tweaks to current stuff relating to the prefab. Also want to get some prototyping done for *something* I've been working on. Might use that instead of dropping it in the belt. Depends if I can get it done or not :shelteryarr:
Improved translation/translation mechanic

**Pull request closed without merging**